library-system/src/App.js

```diff
@@ -0,0 +1,60 @@
+// src/App.js
+import React, { useState } from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
```

Comment:
In v6.4, new routers were introduced that support the new data APIs, should have used createBrowserRouter according to the hint